db/migrate/20221121165352_add_type_field_to_events_reminders.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class AddTypeFieldToEventsReminders < ActiveRecord::Migration[7.0]
+  def up
+    DiscoursePostEvent::Event.where.not(reminders: nil).find_each do |event|
+      refactored_reminders = []
+      event.reminders.split(',') do |reminder|
+        refactored_reminders.push(reminder.prepend("notification."))
+      end
+      event.reminders = refactored_reminders.join(',')
+      event.save
+    end
+  end
+
+  def down
```

Comment:
I don't think this is necessary, we should raise `ActiveRecord::IrreversibleMigration` here.